cli/tests/test_alias.rs

```diff
@@ -150,9 +150,9 @@ fn test_alias_calls_help() {
         output.normalize_stdout_with(|s| s.split_inclusive('\n').take(5).collect()), @r"
     Jujutsu (An experimental VCS)
 
-    To get started, see the tutorial at https://jj-vcs.github.io/jj/latest/tutorial/.
+    To get started, see the tutorial [`jj help -k tutorial`]
 
-    Usage: jj [OPTIONS] <COMMAND>
```

Comment:
Should I change the number of lines for this test, so that the line `Usage: ...` stays in the snapshot?